### PR TITLE
Paginated and cleaned frontend

### PR DESCRIPTION
**Pull request by blacksph3re**
I have removed a lot of unnecessary code and tried to fix some whitespace issues. Also the circles and users endpoints are now paginated, bodies unfortunately returns a weird 403

### CODE REVIEW

**Comment by i3anaan** [comment]
@blacksph3re (Feel free to not answer immediately)
Did you solve the 403 error yet? Or do you know what needs to be fixed? (Was it related to the role entries being deleted/renamed?)

File: docker/omscore-nginx/nginx.conf [hunk 1]
```diff
@@ -11,7 +11,7 @@ events {
 
 http {
   server_tokens off;
-  sendfile on;
+  sendfile off;
```
Comment (i3anaan) [inline docker/omscore-nginx/nginx.conf:14]: Did you have a specific reason to turn this off?

**Review by i3anaan — CHANGES_REQUESTED** [review]
Is not working for me :(

screenshot-localhost-2017-09-20-20-01-13-504

**Comment by i3anaan** [comment]
Correction: I did a full wipe of docker stuff and now it is working, looks really good!
screenshot-localhost-2017-09-20-20-20-08-678

**Review by i3anaan — APPROVED** [review]

File: docker/omscore-nginx/nginx.conf (hunk 1 above)
Comment (blacksph3re) [reply docker/omscore-nginx/nginx.conf:14]: Yes it's creating troubles in docker containers and is merely a performance optimization of nginx

**Pull request merged**